run.py

```diff
@@ -23,8 +23,8 @@ def main():
     # root = Path("renders/real_highway")
     root = Path("renders/ProjectiveSyntheticPedestrians")
 
-    renderer = PreviewRenderer(bpy.context, root)
-    # renderer = Renderer(bpy.context, root)
+    # renderer = PreviewRenderer(bpy.context, root)
```

Comment:
We should decide on a default value for the run file. I assume the PreviewRenderer is a good candidate for that. What do you think?
If you agree we should keep that, if not we should not activate it as default in the other pull request.